src/templates/gene.html

```diff
@@ -36,19 +35,61 @@ <h6 class="m-0 font-weight-bold text-primary">Information</h6>
                                 {% for f in fields %}
                                 {% set param = {f: variant[f]} %}
                                 <td>
+                                    <!-- Items with links -->
                                     {% if f == "dn_no" %}
                                     <a href={{ url_for('patient', id=variant[f]) }}>{{ variant[f] }}</a>
                                     {% elif f == "gene" %}
                                     <a href={{ url_for('gene', id=variant[f]) }}>{{ variant[f] }}</a>
                                     {% elif f == "fullgnomen" %}
-                                    <a href={{ url_for('variant', id=variant[f]) }}>{{ variant[f] }}</a>
+                                        {% for key, value in variant["platformDatasets"].items() %}
```

Comment:
1. I had the same feeling and thought about this as well. Before there was less formatting in HTML due to two reasons. One is that there was a lot less data available in the Excel and secondly the data was more "linear" in excel (except for the metadata above the header, everything was row based). I can avoid this by formatting everything before import (import_data.py), but then I just move the "hardcoded" stuff and make it less flexible for the future. If I need to change something or big parts are added, I need to change it in import_data.py and possibly have to reupload the whole database.
2. I did check the format using autopep8 and HTML extensions in my editor. I did indent a few for loops to make it more readable and personal preference
3. There is a lot at css already and I think you might have a point here. however not everything should be put in css to avoid having "invincible" settings that are hard to change. I do not have the right answer now, but I think improvements can be made here. however if possible I would like to do this at a later stage. Made an issue #14